app/src/main/res/layout-land/reading_text_size_fragment.xml

```diff
@@ -2,6 +2,13 @@
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
You can remove `mobile-landscape` file as only the bottom padding of recyclerview is different and rest everything is same.

Reply:
done

Reply:
What I meant with this is that you can create a dimen value for bottom padding in `values/dimens.xml` and `values-land/dimens.xml` can set different values for this as per mocks and then use this value in `layout/reading_text_size_fragment.xml` file.

Reply:
done